src/lib/eina/eina_inline_lock_posix.x

```diff
@@ -19,17 +19,12 @@
 #ifndef EINA_INLINE_LOCK_POSIX_X_
 #define EINA_INLINE_LOCK_POSIX_X_
 
-#ifdef _XOPEN_SOURCE
-# define EINA_XOPEN_SOURCE _XOPEN_SOURCE
-# undef _XOPEN_SOURCE
-#endif
-#define _XOPEN_SOURCE 600
+#include <semaphore.h>
```

Comment:
did you removed semaphore.h?

Reply:
Sorry, what do you mean? semaphore.h should exist at Linux systems.